src/app/common/transactions/bitcoin/utils.ts

```diff
@@ -22,8 +22,7 @@ export const getColorFromBitcoinTx = (tx: BitcoinTx) => {
     pending: 'warning.label',
     success: 'stacks',
   };
-
-  return colorMap[statusFromBitcoinTx(tx)] ?? 'feedback-error';
+  return colorMap[statusFromBitcoinTx(tx)] ?? 'error.label';
```

Comment:
I'm going to refactor this to remove this fallback as it seems we don't ever hit it as per: https://github.com/leather-wallet/extension/pull/4636#discussion_r1414163139 

If we do need to show this error let me know